queries.sql

```diff
@@ -8,4 +8,60 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 'yes';
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN TRANSACTION;
+ 
+  UPDATE animals SET species = 'unspecified';
+ 
+ROLLBACK TRANSACTION;
+
```

Comment:
- [ ] You should verify the changes before and after the rollback as requested in the project requirements

> Verify that change was made and persists after commit.
> After the roll back verify if all records in the animals table still exist. After that you can start breath as usual ;)

Reply:
Aha, Sure. I will verify all changes before and after rollback.